accessibility/compliance.md

```diff
@@ -5,35 +5,140 @@ description: Compliance with the accessibility standards and requirements in the
-
-This article will be updated with details on the Section 508 and WCAG 2.2 compatibility levels.
-
-<!--
->caption Accessibility compliance levels support provided by the Telerik UI for Blazor components.
--->
-
-
-
-
-
-
-The Telerik UI for Blazor components are highly extensible and customizable. This means that, depending on the level of customization applied, you may be introducing rendering that is not accessible. Therefore, it is recommended that you test any modifications and templates you create to ensure the components still meet the desired level of accessibility standards. Additionally, be mindful of components working with custom input (images, text, HTML content, and so on) and make sure your content is accessible, too.
-
-
+This article lists the accessibility compliance of the Telerik UI for Blazor components.
+
+## Accessibility Conformance Report
+
+The [Accessibility Conformance Report (ACR)](https://www.section508.gov/sell/acr/) is a document that explains how information and communication technology products such as software, hardware, electronic content, and support documentation conform to leading global accessibility standards. Telerik UI for Blazor provides an ACR through the <a href="https://www.itic.org/policy/accessibility/vpat" target="_blank">Voluntary Product Accessibility Template (VPAT®)</a>.
+
+>tip Download the latest version of the <a href="assets/BlazorVPAT.doc" download>Telerik UI for Blazor Accessibility Conformance Report</a>.
+
+## Compliance Table
+
+The table below specifies the level of WCAG 2.2 compliance of each Telerik component for Blazor. The table also provides links to information about keyboard shortcuts, WAI-ARIA attributes, and additional component-specific accessibility documentation.
+
+The information is subject to the following considerations:
+
+* All components implement the required WAI-ARIA attributes without the need for any extra configuration. Some components may provide parameters that render additional optional WAI-ARIA attributes, for example, `aria-label` or `aria-describedby`.
+* The compliance levels are achievable with the [*Default Ocean Blue A11y* theme swatch]({%slug accessibility-overview%}#color-contrast).
+* The accessibility and compliance of some components may depend on the enabled features. In such cases, the compliance level below is based on the default component configuration.
+* Component templates introduce custom markup that may not be accessible. Test any modifications to ensure the web content still meets the desired level of accessibility compliance. Be mindful of components that work with user input such as images, text, or HTML content.
+* Due to the complexity of some components, there are scenarios that are not covered by the WAI-ARIA specification. In those cases, we benefit from the know-how of the dedicated accessibility professionals at Progress Software and feedback from accessibility-minded users.
```

Comment:
What does this mean for the Telerik clients? How does it affect them?

Reply:
These are mostly disclaimers, which imply that the accessibility compliance in the customer's app may not match what we claim in the table.